.github/workflows/tools-api-docs-release.yml

```diff
@@ -9,8 +9,9 @@ jobs:
     runs-on: ubuntu-18.04
     strategy:
       matrix:
-        dir: [latest, $GITHUB_REF]
-
+        dir: 
+            - latest
+            - ${{ github.ref }}
```

Comment:
This is meant to be just the release name, right? I think that the ref gives something like `refs/tags/0.3.1` or similar.

In the past I've used `${{ github.event.release.tag_name }}` for this. eg. 

https://github.com/nf-core/tools/blob/7721f3742fc30a9e6e46d879404bd1366345092d/.github/workflows/push_dockerhub_release.yml#L32

Reply:
This is true, I thought we would only get `0.3.1` but I misread the docs. Either way, `github.event.release.tag_name` shows more clearly what we are trying to achieve.